en/docs/learn/working-with-acr-and-amr.md

```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
 	
 	Note that the AMR values in this example are 	`DemoFaceIdAuthenticator`	, 	`BasicAuthenticator`	, and 	`DemoFingerprintAuthenticator`. These are the authenticators that are used in the authentication process.
 
+#### Translating AMR values
+Follow the steps below to translate the AMR values that get returned to the authentication process.
+
+1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf directory`.
+
+2. Add the following configurations
+
+    !!! note 
+    You can add any custom AMR mapping to this config.
+
+       ```
+       [[authentication_context.method_refs]]
+       method = "SMSOTP"
+       amr_value = "sms"
+       [[authentication_context.method_refs]]
+       method = "EmailOTP"
+       amr_value = "otp"
+       [[authentication_context.method_refs]]
+       excluded_methods = ["BasicAuthenticator", "AuthenticationMethodToBeIgnored"]
+       ```
+	
+    `method`: Authentication Method Name
```

Comment:
In bullet points:

- `method`: Name of the authentication method.
- `amr_value`: The AMR value mapping you prefer to have as the AMR in the ID token. See [AMR draft specification](https://tools.ietf.org/html/draft-ietf-oauth-amr-values-04) for details.
- `excluded_methods`: The list of authentication methods that you would like to exclude from the ID token.

Reply:
Fix with a5c790700af9084435520e134ba8f71368868655